packages/client/hmi-client/src/components/workflow/scenario-templates/situational-awareness/situational-awareness-scenario.ts

```diff
@@ -121,6 +149,52 @@ export class SituationalAwarenessScenario extends BaseScenario {
 			{ x: 0, y: 0 }
 		]);
 
+		// 3. Setting node states/outputs
+		wf.updateNode(modelNode, {
+			state: {
+				modelId: this.modelSpec.id
+			},
+			outputValue: this.modelSpec.id
+		});
+
+		wf.updateNode(datasetNode, {
+			state: {
+				datasetId: this.datasetSpec.id
+			},
+			outputValue: this.datasetSpec.id
+		});
+
+		wf.updateNode(modelConfigNode, {
+			state: {
+				transientModelConfig: modelConfig
+			},
+			outputValue: this.modelConfigSpec.id
+		});
+
+		let calibrateChartSettings: ChartSetting[] = [];
+		calibrateChartSettings = updateChartSettingsBySelectedVariables(
+			calibrateChartSettings,
+			ChartSettingType.VARIABLE,
+			this.calibrateSpec.ids
+		);
+		calibrateChartSettings = updateChartSettingsBySelectedVariables(
+			calibrateChartSettings,
+			ChartSettingType.ERROR_DISTRIBUTION,
+			this.calibrateSpec.ids
+		);
+		calibrateChartSettings = updateChartSettingsBySelectedVariables(
+			calibrateChartSettings,
+			ChartSettingType.INTERVENTION,
+			this.calibrateSpec.ids
+		);
```

Comment:
```suggestion
```
You probably don't need to add intervention chart settings. If the template doesn't create interventions.